src/PresentationalComponents/Common/Tables.js

```diff
@@ -49,9 +49,42 @@ export const buildTagFilter = (tagFilters) => {
   };
 };
 
+const mapUpdateMethodFilterToAPISpec = (filters) => {
+  //If none option is chosen
+  if (filters?.update_method?.includes('none')) {
+    //remove update_method filter from API options and set impacting to false if only none options is chosen
+    if (filters.update_method === 'none') {
+      delete filters.update_method;
+      filters.impacting = 'false';
+
+      //in any other cases, remove 'none' option from the API options as it does have any handler
+      //concatenate or set false to impacting API option
+    } else {
+      const filteredValues = filters.update_method.replace(
+        /,?none||none,?/g,
+        ''
+      );
+
+      filters.update_method = filteredValues;
+      filters.impacting =
+        filters.impacting.toString().length > 1
+          ? filters.impacting.concat(',false')
+          : 'false';
+    }
+  }
+
+  //when user deselects all update_method filters remove the both update_method and impacting filters
+  if (filters?.update_method === '') {
+    delete filters.update_method;
+    delete filters.impacting;
+  }
```

Comment:
I'd put it in an `else if` instead of an `if`.
If the `if` condition above this one would get refactored and the update_method changed to `''` it could trigger this condition and it looks separated this way

Reply:
hmm, this makes sense. Will add it. Thank you!